spec/spec_helper.rb

```diff
@@ -1,9 +1,5 @@
 require 'rails/all'
 
-class << RSpec
-  attr_writer :configuration
-end
-
```

Comment:
Nice clean up of old cruft. Looks like this got included as a private API in rspec/rspec-core#1808